### DISCUSSION TITLE
How to use an external $ref in __get_pydantic_json_schema__?

### DISCUSSION THREAD

**Discussion by Holt59** [pydantic/pydantic#6728] [Pydantic V2]
The following fails because Pydantic tries to resolve a local `$ref` while I want to use an external one:

```python
@classmethod
def __get_pydantic_json_schema__(
    cls, core_schema: core_schema.CoreSchema, handler: GetJsonSchemaHandler
) -> JsonSchemaValue:
    return {"type": "object", "$ref": "https://geojson.org/schema/GeoJSON.json"}
```

Is there a way to tell Pydantic to not touch that `$ref`?

**Comment by glopesdev** [comment]
@Holt59 Seems related to https://github.com/pydantic/pydantic/issues/892, there might be a few good solutions for your external URL case.
